noxfile.py

```diff
@@ -178,35 +141,15 @@ def set_dev(session):
     # is fixed
     session.run(python, "-m", "pip", "install", "setuptools", external=True)
     if sys.platform == "linux":
```

Comment:
This platform-specific check should not be needed anymore

Reply:
JAX is fine on windows then? I left it because it was unrelated to ODES

Reply:
Yes, JAX works on Windows

Reply:
See above, I think that should be a different PR